Applications/Spire/Source/Ui/OverlayPanel.cpp

```diff
@@ -78,15 +88,15 @@ bool OverlayPanel::is_closed_on_blur() const {
 }
 
 void OverlayPanel::set_closed_on_blur(bool is_closed_on_blur) {
```

Comment:
The spec has been updated to reflect that it's close on focus out rather than on blur.